extensions/amp-mowplayer/0.1/test/test-amp-mowplayer.js

```diff
@@ -19,9 +19,8 @@ import {Services} from '../../../../src/services';
 import {VideoEvents} from '../../../../src/video-interface';
 import {listenOncePromise} from '../../../../src/event-helper';
 
-const EXAMPLE_VIDEOID = 'myfwarfx4tb';
-const EXAMPLE_VIDEOID_URL =
-  'https://cdn.mowplayer.com/player.html?code=myfwarfx4tb';
+const EXAMPLE_VIDEOID = 'v-myfwarfx4tb';
```

Comment:
Could you please update the validator tests in `validator-amp-mowplayer.html` with this video ID and re-run validator tests using `gulp validator --update_tests`?